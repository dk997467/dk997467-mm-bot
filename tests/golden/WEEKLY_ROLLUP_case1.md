WEEKLY SOAK ROLLUP

| metric | v1 | v2 | v3 |
|--------|----|----|----|
| edge_net_bps | 2.800000 | 2.750000 | 2.850000 |
| order_age_p95_ms | 305.000000 | 315.000000 |  |
| taker_share_pct | 12.500000 | 13.500000 |  |
| ledger_equity_change_eur | 0.700000 | fees=0.700000 | rebates=-0.350000 |

Verdict: GO


