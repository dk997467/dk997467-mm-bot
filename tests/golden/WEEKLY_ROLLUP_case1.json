{"edge_net_bps":{"median":2.8,"p25":2.7,"p75":2.85,"trend_vs_prev_week_bps":0.0},"ledger":{"equity_change_eur":0.5,"fees_eur":0.6000000000000001,"rebates_eur":-0.30000000000000004},"order_age_p95_ms":{"median":305.0,"p90":320.0,"trend_vs_prev_week_ms":0.0},"period":{"from":"2025-01-02","to":"2025-01-08"},"regress_guard":{"edge_ok":true,"latency_ok":true,"taker_ok":true,"trend_ok":true},"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"taker_share_pct":{"median":12.5,"p90":14.0,"trend_vs_prev_week_pct":0.0},"verdict":"GO"}
